test/lit/passes/inlining-optimizing_optimize-level=3.wast

```diff
@@ -4065,8 +4065,8 @@
  ;; CHECK-NEXT:  (local $21 i32)
  ;; CHECK-NEXT:  (local $22 i32)
  ;; CHECK-NEXT:  (local $23 i32)
- ;; CHECK-NEXT:  (local $24 f64)
- ;; CHECK-NEXT:  (local $25 i32)
+ ;; CHECK-NEXT:  (local $24 i32)
+ ;; CHECK-NEXT:  (local $25 f64)
```

Comment:
Yeah, sorry about that - for a change like this, inlining more things, the diff on this existing file is quite large.

(Ideally we could somehow make this particular file ignore this change, but otherwise I guess ignoring it in review is second best.)